### PR TITLE
Add the upgrade menu entry

### PR DESCRIPTION
**Pull request by rsalevsky**

### CODE REVIEW

**Comment by vuntz** [comment]
Thanks. Let's only merge this once we're ready to push the maintenance update that allows upgrading.

**Comment by rsalevsky** [comment]
Next update will include the Upgrade so let's merge it.

File: crowbar_framework/config/navigation.rb [hunk 1]
```diff
@@ -38,6 +38,7 @@
       level2.item :queue, t("nav.utils.queue"), deployment_queue_path
       level2.item :repositories, t("nav.utils.repositories"), repositories_path
       level2.item :backup, t("nav.utils.backup"), backups_path
+      level2.item :backup, t("nav.utils.upgrade"), "/upgrade"
```
Comment (skazi0) [inline crowbar_framework/config/navigation.rb]: Just noticed. Shouldn't it be `:upgrade` instead of `:backup` here?

**Comment by skazi0** [comment]
[(2) S58P5 Add the "Upgrade" link from Crowbar UI](https://trello.com/c/cdtWLx6h/391-2-s58p5-add-the-upgrade-link-from-crowbar-ui)

**Pull request merged**